com.rustdesk.RustDesk.metainfo.xml

```diff
@@ -28,16 +30,31 @@
   </categories>
   <screenshots>
     <screenshot type="default">
+      <caption>Remote desktop session</caption>
       <image>https://user-images.githubusercontent.com/71636191/171661982-430285f0-2e12-4b1d-9957-4a58e375304d.png</image>
     </screenshot>
   </screenshots>
+  <branding>
+    <color type="primary" scheme_preference="light">#00bbe3</color>
+    <color type="primary" scheme_preference="dark">#0176fa</color>
+  </branding>
   <url type="homepage">https://rustdesk.com</url>
   <url type="bugtracker">https://github.com/rustdesk/rustdesk/issues</url>
-  <url type="donation">https://github.com/sponsors/rustdesk</url>
-  <url type="help">https://rustdesk.com/docs/en/technical-support/</url>
+  <url type="faq">https://github.com/rustdesk/rustdesk/wiki/FAQ</url>
+  <url type="help">https://rustdesk.com/docs</url>
+  <url type="donation">https://ko-fi.com/rustdesk</url>
+  <url type="vcs-browser">https://github.com/rustdesk/rustdesk</url>
   <url type="translate">https://github.com/rustdesk/rustdesk/tree/master/src/lang</url>
   <url type="contribute">https://github.com/rustdesk/rustdesk/blob/master/docs/CONTRIBUTING.md</url>
-  <url type="vcs-browser">https://github.com/rustdesk/rustdesk</url>
+  <url type="contact">https://rustdesk.com/docs/en/technical-support</url>
+  <requires>
+    <display_length compare="ge">600</display_length>
+    <internet>always</internet>
+  </requires>
```

Comment:
I understand that these are logical things to set, but I'm wondering, do you get these values from the specific places in the rustdesk repository, or are they just your inner feelings?

Reply:
If RustDesk had specified a minimum window width I'd have gone with that. It doesn't, though, so I tried to make a very rough estimate of the smallest width the app is guaranteed to work at without any UI elements being cropped away.

From some quick experimentation, I found that the "one-time password length" preference in the security settings is the most width-constrained element in the app. With a window size smaller than 600px the "10" option would be cropped. My inner feeling is that that would be undesirable :slightly_smiling_face: 

![Skjermbilde fra 2024-10-09 13-17-22](https://github.com/user-attachments/assets/63b28d74-4687-4c7d-b39a-03dd80522d77)

As for the [`<internet>`](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-relations-internet) tag, `always` might sound like a little much, but the only other alternatives are `offline-only` and `first-run`. It being in `<required>` means that the app needs an internet connection to be useful. An alternative would be to do this instead:

```xml
<recommends>
    <internet>always</internet>
</recommends>
```

or this:

```xml
<supports>
   <internet>offline-only</internet>
<supports>
```

If you think one of those is a better description of the relationship between internet access and usability in RustDesk, it can be changed.

Reply:
@rustdesk, I hope you are satisfied with these new requirement tags. They appear to be logical and justified.

Reply:
ok for me